guides/plugins/plugins/administration/using-assets.md

```diff
@@ -0,0 +1,86 @@
+# Using assets
+
+## Overview
+
+When working with an own plugin, the usage of own custom images or other assets is a natural requirement. 
+So of course you can do that in Shopware as well. In this guide we will explore how you can
+add and use custom assets in your plugin in order to use them in the administration.
+
+## Prerequisites
+
+In order to be able to start with this guide, you need to have an own plugin running. As to most guides, this guide
+is also built upon the Plugin base guide:
+
+{% page-ref page="../plugin-base-guide.md" %}
+
+Needless to say, you should have your image or another asset at hand to work with.
+
+## Using custom assets
+
+In order to add your own custom assets, you need to save your assets in `Resources/app/administration/static`folder.
+```bash
+# PluginRoot
+.
+├── composer.json
+└── src
+    ├── Resources
+    │   ├── app
+    │       └── administration
+    │             └── static
+    │                   └── your-image.png <-- Asset file here
+    └── SwagBasicExample.php
+```
+
+Similar as in [using custom assets in Storefront](./../storefront/add-custom-assets.md), you need to execute the 
+following command:
+```bash
+bin/console assets:install
+```
+
+This way, your plugin assets are copied to the `public/bundles` folder:
+
+```
+# shopware-root/public/bundles
+.
+├── administration
+├── framework
+├── storefront
+└── SwagBasicExample
+    └── your-image.png <-- Your asset is copied here
+```
+
+### Use assets in the administration
+
+After adding your assets to the `public/bundles` folder, you can start using your assets in the administration. 
+Basically, you just need to use the Vue [filter](https://vuejs.org/v2/guide/filters.html) `asset` we introduce 
+via a `factory.
+```html
+<img :src="'/<plugin root>/static/your-image.png' | asset">
+```
+
+To mention a simple example, we override the block `sw_dashboard_index_content_intro_card` of the dashboard module in 
+`<plugin root>/src/Resources/app/administration/src/main.js` by adding a custom asset.
+
+{% code title="<plugin root>/src/Resources/app/administration/src/main.js" %}
+```js
+Shopware.Component.override('sw-dashboard-index', {
```

Comment:
Imho we can shorten this example. I have just done it this way in a test plugin. But I guess for the documentation this would be fine:

```html
<img :src="'/<plugin root>/static/your-image.png' | asset">
```

The other override stuff etc. creates too much noise and it should be clear that this can be done in the twig/html.

Reply:
@PaddyS What do you think? Should we also have a "running" example or is the html snippet enough?

Reply:
I removed the example for now and if desired, I'll add it anew.